src/py42/clients/securitydata.py

```diff
@@ -307,8 +296,37 @@ def _get_other_file_location_version(self, md5_hash, sha256_hash):
             version = self._preservation_data_service.find_file_version(
                 md5_hash, sha256_hash, paths
             )
-            if version.status_code != 204:
-                return version
+            if version.status_code == 200:
+                return version.data
+
+    def _get_file_stream(self, version):
+        if version.get(u"edsUrl"):
```

Comment:
if the version found was in a `securityEventVersions` prop, it will have an `edsUrl` property, otherwise the prop will be named `storageNodeURL`.